package.json

```diff
@@ -17,16 +17,22 @@
     "license": "ISC",
```

Comment:
I tried to run npm run lint, but it fall with errors.
Could you please check if it runs successfully on your local machine?